src/AWS.Deploy.Recipes/RecipeDefinitions/ASP.NETAppAppRunner.recipe

```diff
@@ -100,6 +100,7 @@
             "TypeHint": "AppRunnerService",
             "AdvancedSetting": false,
             "Updatable": false,
+            "VisibleOnRedeployment": true,
```

Comment:
Throughout, I'd probably be a little more conservative and leave anywhere we're displaying an ID/ARN/etc. visible if possible. In case a user is using this as a reference and then doing something else with the associated resources in the console or on another screen.
* AppRunner - KMS key
* Fargate - VPC Id, Load Balancer ARN
* Beanstalk - IAM role ARN, EC2 key pair

Reply:
Since the summary screen shows all the currently selected options, I wanted the edit screen to be mostly the updatable settings which is why I wasn't very generous with visibility of disabled settings. Curious what @normj and @96malhar think? and as you said, changing the visibility of the disabled settings is not a breaking change and we can update based on feedback.

Reply:
As a user, I'd prefer to always see the AWS resources associated with my application across all configuration views (including the edit screen) even if I cannot modify them. 

To be explicit about the immutability of certain settings on a re-deployment, we should append a line to the OptionSetting description saying - _"The configured value cannot be changed during a redeployment to a existing Cloud Application"_. This line should be appended wherever `Updatable` is set to `false` and `VisibleOnRedeployment` is set to `true`

Reply:
I have made the resources that Alex mentioned visible. Also, I have experimented with adding a message to the description but it looked weird and repetetive in the toolkit. This message should be added by the Toolkit and should look different than the description so it could stand out.